package.json

```diff
@@ -93,7 +93,7 @@
     "multiselect": "~0.9.12",
     "number_helpers": "^0.1.1",
     "patternfly": "^3.58.0",
-    "patternfly-react": "^2.34.1",
+    "patternfly-react": "^2.36.1",
```

Comment:
the switcher wasn't exported in pf's `index.js` in the original PR.  it was fixed since then so we need a bump